javascript/builtins/Function.json

```diff
@@ -122,10 +122,10 @@
                 "version_added": false
               },
               "firefox": {
-                "version_added": false
+                "version_added": "1"
               },
               "firefox_android": {
-                "version_added": false
+                "version_added": "4"
```

Comment:
This looks weird; can you confirm it is correct (I don't think so, at least not without a version_removed I guess)

Reply:
Yep, same here, changed it back to false.